### DISCUSSION TITLE
Installing Chevreto 4.0.7 - No install folder

### DISCUSSION THREAD

**Discussion by arnehulstein** [chevereto/chevereto#11] [Q&A]
I tried to install the Chevreto package as downloaded from Github. It runs on a cpanel installation, so I was following the installation instructions at v4docs but then it tells me to go to the location and add /install but there is no install folder in the package, so this does not work.
Is there anywhere I can find that package, or can I find the /install anywhere?

Thanks!

**Comment by rodber** [comment]
> it tells me to go to the location and add /install but there is no install folder in the package, so this does not work.

Chevereto requires virtual url setup: https://v4-docs.chevereto.com/application/stack/web-server.html
